### PR TITLE
Move to global configs and cleanup code

### PR DESCRIPTION
**Pull request by skjnldsv**
This app is still only used for the public pages, we can kill it when https://github.com/nextcloud/viewer/issues/644 is done

This should solve most of the dependencies madness that we have here currently! :see_no_evil:

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -0,0 +1,201 @@
+/**
+ * @copyright 2018 Christoph Wurst <christoph@winzerhof-wurst.at>
```
Comment (PVince81) [inline src/main.js:2]: where does this file come from ?

adding a new big file like these sounds more than just "cleanup code" :smile:

Comment (skjnldsv) [reply src/main.js:2]: Just renaming from the old 'viewer.js' file

**Review by PVince81 — APPROVED** [review]
:+1: looks alright, I haven't tested though

**Review by juliushaertl — APPROVED** [review]

**Pull request merged**